ui/tests/acceptance/auth-test.js

```diff
@@ -202,10 +208,85 @@ module('Acceptance | auth', function (hooks) {
       () => new Error('should not call renew-self directly after logging in')
     );
 
-    await visit('/vault/auth');
-    await fillIn(AUTH_FORM.method, 'token');
-    await fillIn(AUTH_FORM.token, rootToken);
-    await click('[data-test-auth-submit]');
+    await login(rootToken);
     assert.strictEqual(currentURL(), '/vault/dashboard');
   });
+
+  module('Enterprise', function (hooks) {
+    hooks.beforeEach(async function () {
+      const uid = uuidv4();
+      this.ns = `admin-${uid}`;
+      // log in to root to create namespace
+      await login();
+      await runCmd(createNS(this.ns), false);
+      // login to namespace, mount userpass, create policy and user
+      await loginNs(this.ns);
+      this.db = `database-${uid}`;
+      this.userpass = `userpass-${uid}`;
+      this.user = 'bob';
+      this.policyName = `policy-${this.userpass}`;
+      this.policy = `
+        path "${this.db}/" {
+          capabilities = ["list"]
+        }
+        path "${this.db}/roles" {
+          capabilities = ["read","list"]
+        }
+        `;
+      await runCmd([
+        mountAuthCmd('userpass', this.userpass),
+        mountEngineCmd('database', this.db),
+        createPolicyCmd(this.policyName, this.policy),
+        `write auth/${this.userpass}/users/${this.user} password=${this.user} token_policies=${this.policyName}`,
+      ]);
+      return await logout();
+    });
+
+    hooks.afterEach(async function () {
+      await visit(`/vault/logout?namespace=${this.ns}`);
+      await fillIn(AUTH_FORM.namespaceInput, ''); // clear login form namespace input
+      await login();
+      await runCmd([`delete sys/namespaces/${this.ns}`], false);
+    });
+
+    // this test is specifically to cover a token renewal bug within namespaces
```

Comment:
totally not required, but i've found it helpful when folks have mentioned that a test is written for a specific bug to include the github pr or issue link in the comment. Matthew has done this a couple times and I've used those links for more context or additional regressions testing.

Reply:
I was going to do this if it was a GitHub link! But since it's a jira link I didn't include it. I remember a while ago the company discussed that we shouldn't share internal links publicly.

Reply:
ah, yep that's the correct move then.